### PR TITLE
chore(repo) add gitignore

### PR DESCRIPTION
**Pull request by sealet**
@seclay2 

git ignore for unwanted files

### CODE REVIEW

**Comment by sealet** [comment]
@seclay2

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+**jacoco.exec
+
+*.orig
+*.iws
+*.iml
+*.ipr
+*.settings
+*.log
+*.mov
```
Comment (seclay2) [inline .gitignore]: need .mov

**Review by seclay2 — CHANGES_REQUESTED** [review]

**Comment by sealet** [comment]
@seclay2 removed mov files from the ignore list

**Review by seclay2 — APPROVED** [review]
👍

**Pull request merged**